Resources/Prototypes/DeltaV/Catalog/VendingMachines/Inventories/pride.yml

```diff
@@ -17,4 +17,16 @@
        ClothingHeadHatXmasCrown: 2
        BedsheetRainbow: 2
        ClothingUniformColorRainbow: 2
-       ClothingUnderSocksCoder: 1
\ No newline at end of file
+       ClothingUnderSocksCoder: 2
+       ClothingUnderSocksStripedWhiteandLightBlue: 2 # DeltaV - Neptune's striped socks
+       ClothingUnderSocksStripedWhiteandGreen: 2 # DeltaV - Neptune's striped socks
+       ClothingUnderSocksStripedWhiteandLightBlueBunnySlipper: 2 # DeltaV - Neptune's striped socks
+       ClothingUnderSocksStripedWhiteandOrange: 2 # DeltaV - Neptune's striped socks
+       ClothingUnderSocksStripedWhiteandPurple: 2 # DeltaV - Neptune's striped socks
+       ClothingUnderSocksStripedWhiteandRed: 2 # DeltaV - Neptune's striped socks
+       ClothingUnderSocksStripedWhiteandYellow: 2 # DeltaV - Neptune's striped socks
+       ClothingUnderSocksStripedYellowandPurple: 2 # DeltaV - Neptune's striped socks
+       ClothingUnderSocksStripedCaptainBlueandGold # DeltaV - Neptune's striped socks
+       ClothingUnderSocksStripedCaptainBlueandWhite # DeltaV - Neptune's striped socks
+       ClothingUnderSocksStripedCaptainGoldandBlue # DeltaV - Neptune's striped socks 
```

Comment:
Line 29, 30 and 31 are all missing correct formatting, which is what's causing your error